### PR TITLE
Remove retina endpoints

### PR DESCRIPTION
**Pull request by jmsmkn**
See https://github.com/DIAGNijmegen/rse-grand-challenge-admin/issues/236

### CODE REVIEW

**Comment by jmsmkn** [comment]
The tests on main are likely broken.

File: tests/async_integration_tests.py [hunk 1]
```diff
@@ -170,20 +73,22 @@ async def test_chunked_uploads(local_grand_challenge):
...
-    async with AsyncClient(
-        token=RETINA_TOKEN, base_url=local_grand_challenge, verify=False
-    ) as c_retina:
-        existing_chunks_retina = (await c_retina(path="uploads/"))["count"]
+    # archive
+    async with (
+        AsyncClient(
+            token=ARCHIVE_TOKEN, base_url=local_grand_challenge, verify=False
+        ) as c_archive
+    ):
```
Comment (HarmvZ) [inline tests/async_integration_tests.py:81]:
```suggestion
    async with AsyncClient(
        token=ARCHIVE_TOKEN, base_url=local_grand_challenge, verify=False
    ) as c_archive:
```

**Review by HarmvZ — CHANGES_REQUESTED** [review]

**Comment by HarmvZ** [comment]
@jmsmkn I just came across this. Do you know what is going on with the failing tests?

**Comment by jmsmkn** [comment]
Likely something with the GC fixtures changed and the tests broke. They should be re-run on main. If so, it is not ideal to rely on those fixtures kept on GC. Maybe we should create the fixtures that are needed for the GCAPI tests within this repo.

**Review by HarmvZ — APPROVED** [review]
Right, I just ran the tests on main and they are also broken. Issue here: https://github.com/DIAGNijmegen/rse-gcapi/issues/155. Let's merge this changeset since the failing tests are unrelated.

**Comment by jmsmkn** [comment]
I'm not sure if this PR works, I think it's better to fix the tests on main first then merge/rebase.

**Comment by HarmvZ** [comment]
Sidenote: Some references to retina stuff are still in the models.py file. This will automatically be removed when we regenerate the file.

**Pull request merged**